kdc/bx509d.8

```diff
@@ -127,23 +127,32 @@ required query parameter.
 In a successful query, the response body will contain a PEM
 encoded end entity certificate and certification chain.
 .Pp
+Or
+.Ar GET
+of
+.Ar /bx509 ,
+as this used to be called.
+.Pp
 Authentication is required.
 Unauthenticated requests will elicit a 401 response.
 .Pp
+Authorization is required.
+Unauthorized requests will elicit a 403 response.
+.Pp
 Subject Alternative Names (SANs) and Extended Key Usage values
 may be requested, both in-band in the CSR as a requested
 extensions attribute, and/or via optional query parameters.
 .Pp
 Supported query parameters (separated by ampersands)
 .Bl -tag -width Ds -offset indent
-.It Li csr = Va <base64-encoded-DER-encoded-CSR>
-.It Li dNSName = Va <hostname>
-.It Li rfc822Name = Va <email-address>
-.It Li xMPPName = Va <XMPP-address>
-.It Li krb5PrincipalName = Va <Kerberos-principal-name>
-.It Li ms-upn = Va <UPN>
-.It Li eku = Va <OID>
-.It Li lifetime = Va <lifetime>
+.It Li csr = Va base64-encoded-DER-encoded-CSR
+.It Li dNSName = Va hostname
+.It Li rfc822Name = Va email-address>
```

Comment:
Should that trailing angle bracket go away?

Reply:
Yes!  Thanks.